src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.Revocation.cs

```diff
@@ -56,14 +111,26 @@ public ValueTask HandleAsync(PrepareRevocationRequestContext context)
                     throw new ArgumentNullException(nameof(context));
                 }
 
-                // Weibo, VK ID and Yandex don't support the standard "token" parameter and
+                // Webflow, Weibo, VK ID and Yandex don't support the standard "token" parameter and
                 // require using the non-standard "access_token" parameter instead.
-                if (context.Registration.ProviderType is ProviderTypes.Weibo or ProviderTypes.VkId or ProviderTypes.Yandex)
+                if (context.Registration.ProviderType is ProviderTypes.Webflow or ProviderTypes.Weibo or ProviderTypes.VkId or ProviderTypes.Yandex)
                 {
                     context.Request.AccessToken = context.Token;
                     context.Request.Token = null;
                     context.Request.TokenTypeHint = null;
                 }
+                
+                // Miro uses a JSON payload that expects the "accessToken", "clientId", and "clientSecret" properties
+                else if (context.Registration.ProviderType is ProviderTypes.Miro)
+                {
+                    context.Request["accessToken"] = context.Token;
+                    context.Request["clientId"] = context.Request.ClientId;
```

Comment:
Interesting, I don't see these parameters mentioned in https://developers.linear.app/docs/oauth/authentication#id-6.-revoke-an-access-token. Are we sure their non-standard revocation endpoint supports client authentication?